### PR TITLE
add testclient app env var to prod

### PR DESCRIPTION
**Pull request by oragame**
**JIRA Ticket:**
N/A

**User Story or Bug Summary:**
Add apps.testclient ENV var to the prod env.yml

### What Does This PR Do?

Adding the testclient env allows this variable to be set on PROD, same as it currently is for IMPL and TEST

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

* syntax looks correct

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? **No**.
    * Does this PR modify or invalidate any of our security controls? **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

* Dependent on removing the duplication of this variable from the web-server repo, both PR's are being worked in tandem to resolve this issue

### Submitter Checklist

I have gone through and verified that...:

* [X] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [ ] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [ ] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [X] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [ ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [X] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [ ] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: vars/env/prod/env.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ env_host_url: "{{ env_external_dns_name }}"
 
 # django environment specific apps list
 # A test client - moved to aws-test / dev /impl settings
-env_django_environment_apps: ['storages', ]
+env_django_environment_apps: ['storages', 'apps.testclient', ]
 env_django_optional_installed_apps: [ ]
```
Comment (njdister) [inline vars/env/prod/env.yml]: This **might** also need added here as well (optional apps).  Not 100% sure but its here for TEST and IMPL too.

Comment (oragame) [reply vars/env/prod/env.yml]: oh... cool. I will match up then

**Review by njdister — APPROVED** [review]

File: vars/env/prod/env.yml [hunk 2]
```diff
@@ -84,8 +84,8 @@ env_host_url: "{{ env_external_dns_name }}"
 
 # django environment specific apps list
 # A test client - moved to aws-test / dev /impl settings
-env_django_environment_apps: ['storages', ]
-env_django_optional_installed_apps: [ ]
+env_django_environment_apps: ['storages', 'apps.testclient', ]
+env_django_optional_installed_apps: ["testclient", ]
```
Comment (JFU-GIT) [inline vars/env/prod/env.yml:88]: should it be apps.testclient on line 88

Comment (njdister) [reply vars/env/prod/env.yml:88]: TEST and IMPL values are the exact same way.  Not sure it matters anymore if we are going to remove the optional apps loading.

Comment (oragame) [reply vars/env/prod/env.yml:88]: I had the same thought, but on TEST and IMPL, the values are exactly like this so I thought it best to leave it alone unless there is an explanation for change?

**Comment by njdister** [comment]
Closing as @dtisza1 is coming up with a fix that does not also require changes to the main web server repo

**Pull request closed without merging**